docs/INSTALL.md

```diff
@@ -1,9 +1,69 @@
 # Building
 
+- [Getting sources](#getting-sources)
 - [Desktop](#desktop-app)
 - [Android](#android-app)
 - [iOS](#ios-app)
 
+## Getting sources
+
+First of all get the source code. The full Organic Maps sources repository is ~8.5Gb in size, however its possible to reduce the size e.g. if full commit history is not required.
+
+For _Windows 10_ enable [symlinks][git-symlinks] support in git first:
+
+[git-symlinks]: https://git-scm.com/docs/git-config#Documentation/git-config.txt-coresymlinks
+
+```bash
+git config --global core.symlinks true
+```
+
+If you plan to contribute and propose pull requests then fork the repo https://github.com/organicmaps/organicmaps first.
```

Comment:
Maybe move this to CONTRIBUTE and reference it here?

Reply:
I don't see a way how to make it elegant and easy to read unless its duplicated...

E.g. imagine in INSTALL we'd have
> If you're planning  to contribute your changes then read CONTRIB first.
> Clone the repo:
> `git clone ....`

And in the CONTRIB
> Fork the repo first and then follow instructions in INSTALL to clone it.

Not good as people will forget to substitute their github name in the clone command.

or
> Fork the repo first and then clone it
> `git clone <forked-repo>...`
> Follow instructions in INSTALL#configure to configure the cloned repo.

Better, but requires a reader to jump back-and-forth between the two docs still.

And if we'll add e.g. options to reduce download size (from below) to CONTRIB and cross-reference them to INSTALL then it'll become more ugly even.

Maybe I just didn't get your idea right though.

Reply:
To avoid duplication, we can write in INSTALL that use this `git clone …` if you just want to build it for yourself and save disk space, or that `git clone…` if you want to contribute.

Reply:
And then start CONTRIBUTING with "Please follow INSTALL.md steps first".